io.otsaloma.nfoview.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.otsaloma.nfoview
+runtime: org.gnome.Platform
+runtime-version: 3.30
+sdk: org.gnome.Sdk
+command: nfoview
+rename-appdata-file: nfoview.appdata.xml
+rename-desktop-file: nfoview.desktop
+rename-icon: nfoview
+copy-icon: true
+finish-args:
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --filesystem=host
```

Comment:
You should use NativeFileChooser (which uses the portal) and remove the host file permission.

Reply:
Assuming you mean [GtkFileChooserNative](https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html), that seems to require GTK+ 3.20 and I'd rather not bump the GTK+ requirement that far yet on account of native distro packages.

Reply:
I'm hardly a real programmer but I find it hard to believe it's impossible to support both cases.

Reply:
From major distributions, only Slackware 14.2 and Debian oldstable ships gtk3 older than gtk 3.20.

Reply:
It is actually likely that I can support both. Generally I don't like that pattern, but with this app the file dialog use is such simple, that it's likely doable. I'll look into it.

Reply:
I just released 1.26, which uses Gtk.FileChooserNative when available and updated this manifest accordingly. I also removed dconf permissions, which I had only added on account the previous direct use of a file chooser dialog, nothing in nfoview itself uses dconf.

https://github.com/otsaloma/nfoview/commit/bad552bb3fb5136fc4d68959f2d8920eb096fe29